packages/graphback-cli/src/helpers/init.ts

```diff
@@ -21,7 +21,7 @@ async function installDependencies(name: string, database: string): Promise<void
   if (database === 'pg') {
     await execa('npm', ['i', '-S', 'pg'])
   } else if (database === 'sqlite3') {
-    await execa('npm', ['i', '-S', 'sqlite3'])
+    await execa('npm', ['i', '-S', 'sqlite3', 'graphql-subscriptions'])
```

Comment:
I think for apollo we might need `apollo-server` as it has in mem implementation. Not 100% sure but I have used it there.

Reply:
We would have to check for template then, because graphql-js would need this package.